refinery/ui/source/js/file-browser/views/files-tab.html

```diff
@@ -65,40 +65,62 @@
         <h3 class="p-l-1">
           {{ FBCtrl.assayFilesTotal }} files
         </h3>
-        <span class="p-l-1"
-          ng-class="{banned: !FBCtrl.dataSet.is_clean}"
-          ng-if="FBCtrl.dataSet.isa_archive.length && FBCtrl.dataSet.is_owner">
-          <a
-            ng-class="{disabledLink: !FBCtrl.dataSet.is_clean}"
-            class="refinery-base btn btn-default btn-xs header-button"
-            ng-href="/data_set_manager/import/?data_set_title={{ FBCtrl.dataSet
-            .title }}/#/isa-tab-import?data_set_uuid={{ FBCtrl.dataSet.uuid }}">
-            Revise Metadata
-          </a>
-        </span>
-        <span class="p-l-1"
-          ng-class="{banned: !FBCtrl.dataSet.is_clean}"
-          ng-if="FBCtrl.dataSet.pre_isa_archive.length && FBCtrl.dataSet.is_owner">
-          <a
-            ng-class="{disabledLink: !FBCtrl.dataSet.is_clean}"
-            class="refinery-base btn btn-default btn-xs header-button"
-            ng-href="/data_set_manager/import/?data_set_title={{ FBCtrl.dataSet
-            .title }}/#/?data_set_uuid={{ FBCtrl.dataSet.uuid }}">
-            Revise Metadata
-          </a>
-        </span>
-        <span class="p-l-1-3" ng-if="FBCtrl.dataSet.is_owner">
-          <a
-            popover-placement="right"
-            uib-popover-html="'Redirects to the Data Set Upload page where you
-            can upload revised metadata. <strong>Note:</strong> Once you have
-            run an analysis or visualization on a data set, its metadata can no longer be revised.'"
-            popover-title="Revise Metadata"
-            popover-trigger="'outsideClick'"
-            popover-append-to-body="true">
-              <i class="fa fa-question-circle info-icon icon-only"></i>
-          </a>
-        </span>
+        <div class="float-right">
+          <h4 class="p-r-1-2">
+            Edit Metadata
+          </h4>
+          <span
+            ng-class="{banned: !FBCtrl.dataSet.is_clean}"
+            ng-if="FBCtrl.dataSet.is_owner">
+            <button
+              ng-class="{disabledLink: !FBCtrl.dataSet.is_clean}"
+              class="refinery-base btn btn-default btn-xs header-button"
+              ng-click="FBCtrl.toggleEditMode()">
+              <span
+                class="disabledLink"
+                ng-show="FBCtrl.editMode">
+                <i class="fa fa-table"></i> Modifying.....
```

Comment:
```suggestion
                <i class="fa fa-table"></i> Modifying...
```

Usual ellipsis is just three dots. (Or `&hellip;`)